api/caching/tasks.py

```diff
@@ -179,8 +179,8 @@ def update_storage_usage_with_size(payload):
         source_provider = payload['source']['provider']
         if target_node == source_node and source_provider == provider:
             return  # Its not going anywhere.
-        if source_provider == 'osfstorage':
-            source_node_usage = source_node.storage_usage
+        if source_provider == 'osfstorage' and not source_node.is_quickfiles:
+            source_node_usage = source_node.storage_usage or 0
```

Comment:
Would it be better to do this at the source? That is, to add `or 0` to the `return`s in `AbstractNode.storage_usage`

Reply:
Good point. I like that better than having to resolve every time.